app/src/test/java/org/simple/clinic/drugs/selection/PrescribedDrugsScreenControllerTest.kt

```diff
@@ -37,100 +43,104 @@ class PrescribedDrugsScreenControllerTest {
 
   @Test
   fun `should correctly construct RecyclerView models from protocol drugs and prescribed drugs`() {
-    val amlodipine = PatientMocker.protocolDrug(name = "Amlodipine", dosages = listOf("5mg", "10mg"))
-    val telmisartan = PatientMocker.protocolDrug(name = "Telmisartan", dosages = listOf("40mg", "80mg"))
-    val chlorthalidone = PatientMocker.protocolDrug(name = "Chlorthalidone", dosages = listOf("12.5mg", "25mg"))
+    val amlodipine5mg = PatientMocker.protocolDrug(name = "Amlodipine", dosage = "5mg")
+    val amlodipine10mg = PatientMocker.protocolDrug(name = "Amlodipine", dosage = "10mg")
+    val telmisartan40mg = PatientMocker.protocolDrug(name = "Telmisartan", dosage = "40mg")
+    val telmisartan80mg = PatientMocker.protocolDrug(name = "Telmisartan", dosage = "80mg")
 
-    val protocol = Protocol(
-        mock(),
-        name = "Dummy protocol",
-        drugs = listOf(amlodipine, telmisartan, chlorthalidone))
-    whenever(protocolRepository.currentProtocol()).thenReturn(Observable.just(protocol))
+    val protocolUuid = UUID.randomUUID()
+    val currentFacility = PatientMocker.facility(protocolUuid = protocolUuid)
+    whenever(userSession.requireLoggedInUser()).thenReturn(Observable.just(PatientMocker.loggedInUser()))
+    whenever(facilityRepository.currentFacility(any<User>())).thenReturn(Observable.just(currentFacility))
+
+    whenever(protocolRepository.drugsForProtocolOrDefault(protocolUuid)).thenReturn(Observable.just(listOf(
+        ProtocolDrugAndDosages(amlodipine10mg.name, listOf(amlodipine5mg, amlodipine10mg)),
+        ProtocolDrugAndDosages(telmisartan40mg.name, listOf(telmisartan40mg, telmisartan80mg))
+    )))
 
     val prescriptionUuid1 = UUID.randomUUID()
     val prescriptionUuid2 = UUID.randomUUID()
     val prescriptionUuid3 = UUID.randomUUID()
     val prescriptionUuid4 = UUID.randomUUID()
 
-    val amlodipinePrescription = PatientMocker.prescription(name = "Amlodipine", dosage = "10mg", isProtocolDrug = true)
-    val telmisartanPrescription = PatientMocker.prescription(uuid = prescriptionUuid1, name = "Telmisartan", dosage = "9000mg", isProtocolDrug = false)
+    val amlodipine10mgPrescription = PatientMocker.prescription(name = "Amlodipine", dosage = "10mg", isProtocolDrug = true)
+    val telmisartan9000mgPrescription = PatientMocker.prescription(uuid = prescriptionUuid1, name = "Telmisartan", dosage = "9000mg", isProtocolDrug = false)
     val ReesesPrescription = PatientMocker.prescription(uuid = prescriptionUuid2, name = "Reese's", dosage = "5 packets", isProtocolDrug = false)
```

Comment:
Capital letter? 😅

Reply:
I'll fix this

Reply:
Assuming this will be fixed in another PR. Resolving this comment now.